crates/types/src/config/networking.rs

```diff
@@ -63,6 +63,20 @@ pub struct NetworkingOptions {
     /// The number of messages that can be queued on the outbound stream of a single
     /// connection.
     pub outbound_queue_length: NonZeroUsize,
+
+    /// # Number of connections between peers
+    ///
+    /// This is used as a guiding value for how many connections every node can
+    /// maintain with its peers. With more connections, concurrency of network message
```

Comment:
Maybe "maintain with each peer" to make it clearer that it's the number of connections per peer. Maybe `num_concurrent_connections_per_peer` (a bit of a mouthful)?

Reply:
I'll update the documentation but I'll probably stick with the short name.